packages/computed/src/computed.ts

```diff
@@ -39,23 +41,76 @@ const DISPOSED_STATE = {
   _options: null
 }
 
-export function computed (evaluatorFunctionOrOptions, evaluatorFunctionTarget, options) {
+export interface Computed<T = any> extends ComputedFunctions<T> {
```

Comment:
Computed can also receive an asymmetrical construction:

```ts
({ write: (v: T) => void, read: () => T }) => this
```

Reply:
I think it is declared in L68
```typescript
export interface ComputedOptions<T = any, TTarget = void> {
  read?: ComputedReadFunction<T, TTarget>;
  write?: ComputedWriteFunction<T, TTarget>;
  owner?: TTarget;
  pure?: boolean;
  deferEvaluation?: boolean;
  disposeWhenNodeIsRemoved?: Node;
  disposeWhen?: () => boolean;
}
```